### ISSUE TITLE
Need to enable a 'Sponsor' button on `open-vsx.org`

### ISSUE THREAD

**Issue by kineticsquid** [EclipseFdn/open-vsx.org#979]
We have a donate button for `open-vsx.org`, details in this issue: https://gitlab.eclipse.org/eclipsefdn/it/websites/eclipse.org/-/issues/120#note_680596. Now we need to provide a link to this donate button page on the main `open-vsx.org` page.

**Comment by kineticsquid** [comment]
@chrisguindon @linkfang Thanks for the quick response on this but...um... I need a slight change. I missed the earlier discussion that we are now using 'Sponsor' instead of 'Donate'. I'm updating the title. I'll also add a comment to the PR. Sorry.

**Comment by kineticsquid** [comment]
@chrisguindon @linkfang I've re-opened this because I think this button got dropped in the process of resolving another issue.

**Comment by chrisguindon** [comment]
@kineticsquid,

I can see that Zhou commit is on the production branch but I don't see the menu link. We will need to investigate.

**Comment by chrisguindon** [comment]
@oliviergoulet5 can you take a look at this issue please?

Zhou tried to add the sponsor link to the menu via https://github.com/EclipseFdn/open-vsx.org/commit/fa939d2339904127a54c3a25ce39bbf80f9d04c4 but we don't see it.

**Comment by chrisguindon** [comment]
@mbarbero can you confirm that the content of https://github.com/EclipseFdn/open-vsx.org/tree/production is what we currently have on production?

@oliviergoulet5 can you test and see if you get the same result if you run the code from the production branch?

**Comment by oliviergoulet5** [comment]
It is working on production as well.

**Comment by chrisguindon** [comment]
I found a note on slack from Mika mentioning that the current image on prod is [ghcr.io/eclipsefdn/openvsx-website:922bf97-49](http://ghcr.io/eclipsefdn/openvsx-website:922bf97-49) which is 5 months old. 

Our changes were created after that time which explain why we don't see them.

@mbarbero In your slack comment, you mention that the image must stay until the fix from @amvanbaren is verified:
https://gitlab.eclipse.org/eclipsefdn/helpdesk/-/issues/1365#note_795228

I can see that the issue is now closed. What else is currently blocking us from publishing a newer image? Is there an issue about this that we can follow?
